WeakAuras/Prototypes.lua

```diff
@@ -9061,7 +9194,6 @@ if WeakAuras.IsClassic() or WeakAuras.IsBCC() then
   if not UnitDetailedThreatSituation then
     Private.event_prototypes["Threat Situation"] = nil
   end
-  Private.event_prototypes["Death Knight Rune"] = nil
```

Comment:
Don't restore this trigger on classic/bcc

Reply:
This one is a little weird at the moment since the `WOW_PROJECT_ID` is still bcc it falls into this condition.  Once there is one this will need to be reworked, or I can leave as nil for the time being.